library/Backends/llvm/src/Juvix/Backends/LLVM/Codegen/Constants.hs

```diff
@@ -0,0 +1,25 @@
+-- | Module for predefined constants
+module Juvix.Backends.LLVM.Codegen.Constants where
+
+import Juvix.Library hiding (Type)
+import qualified LLVM.AST.Type as Type
+
+portLength :: Num p => p
+portLength = 32
+
+-- | An abbreviation for 'IntegerType' 4
+i4 :: Type.Type
+i4 = Type.IntegerType 4
+
+-- | An abbreviation for 'IntegerType' 8
+i8 :: Type.Type
+i8 = Type.IntegerType 8
```

Comment:
We can use `Type.i8` right?

Reply:
We could rexport them from `Type` perhaps. This is more of a constant LLVM type, I think there is a distinction here between Haskell types and LLVM types hence this split.

However I'm fine in breaking it up to have constants in Types.